docs/development.rst

```diff
@@ -106,11 +106,7 @@ are making:
 
     $ git remote add upstream https://github.com/ga4gh/server.git
 
-All development is done against the ``develop`` branch.  The ``stable``
-branch is meant to be kept stable since it is the branch releases are
-based on -- don't touch it!  These are the two mainline branches.
-Our branching model is loosely based on the one described
-`here <http://nvie.com/posts/a-successful-git-branching-model/>`__.
+All development is done against the ``master`` branch.
```

Comment:
I take it we're going to rename the branches after this PR goes into effect?

Also, it would be nice to have the branch name as a sphinx variable instead of hard coded